examples/demo-rollup/README.md

```diff
@@ -74,29 +74,31 @@ circumstances.
 
 ## Getting Started
 
-## Setting up the SDK to run with a local DA layer instance
-In this example, we will spin up a local Celestia instance as our DA layer. To be able to do that:
-* Install docker https://www.docker.com
-* Switch to the `demo-rollup` directory
-* Start the celestia services locally
+### Setting up the SDK to run with a local DA layer instance
+1. Install docker https://www.docker.com.
+2. Switch to the `examples/demo-rollup` directory.
+3. Start the DA layer instance.
+
+
+In this example, we will spin up a local Celestia instance as our DA layer. We've built a small Makefile to simplify that process:
 ```
-make clean
-make start
+$ make clean
+$ make start
 ```
-* Details/Internals of the Makefile are [here](#Makefile)
-* The above command should also configure your local setup so you should see some changes stashed
+If interested, you can check out what the Makefile does [here](#Makefile).
+The above command should also configure your local setup so you should see some changes stashed:
 ```
 $ git status
 ..
 ..
 	modified:   ../const-rollup-config/src/lib.rs
 	modified:   rollup_config.toml
 ```
-* Start the demo-rollup in a different tab
+4. Start the demo-rollup in a different tab:
 ```
-$ cargo run
+$ cargo +nightly run
```

Comment:
We have a `rust-toolchain.toml`, so this is unnecessary and potentially breaking the setup. Let's remove it

Reply:
```suggestion
$ cargo run
```